### PR TITLE
Swap PvP "Attack" with "Walk here" in safe zones.

### PR DESCRIPTION
**Pull request by nateprat**
Swap "Attack" with "Walk here" in PvP Safe Zones 
https://github.com/runelite/runelite/issues/10064

### CODE REVIEW

**Comment by Adam-** [comment]
IMO this seems fine, but I want to wait for the upcoming Jagex statement about the MES before adding anymore stuff to it for now.

File: runelite-client/src/main/java/net/runelite/client/plugins/menuentryswapper/MenuEntrySwapperPlugin.java [hunk 1]
```diff
@@ -33,19 +33,15 @@
 import javax.inject.Inject;
 import lombok.Getter;
 import lombok.Setter;
-import net.runelite.api.Client;
-import net.runelite.api.GameState;
-import net.runelite.api.ItemComposition;
-import net.runelite.api.MenuAction;
-import net.runelite.api.MenuEntry;
-import net.runelite.api.NPC;
+import net.runelite.api.*;
```
Comment (dekvall) [inline runelite-client/src/main/java/net/runelite/client/plugins/menuentryswapper/MenuEntrySwapperPlugin.java]: No [wildcard imports](https://github.com/runelite/runelite/wiki/Code-Conventions).

File: runelite-client/src/main/java/net/runelite/client/plugins/menuentryswapper/MenuEntrySwapperPlugin.java [hunk 2]
```diff
@@ -604,13 +602,30 @@ else if (config.swapBones() && option.equals("bury"))
...
+	{
+		if (WorldType.isPvpWorld(client.getWorldType()))
+		{
+			Widget widget = client.getWidget(PVP_WORLD_SAFE_ZONE);
+			return widget != null && !widget.isSelfHidden();
+		}
+		else
+			{
+				return false;
+			}
```
Comment (dekvall) [inline runelite-client/src/main/java/net/runelite/client/plugins/menuentryswapper/MenuEntrySwapperPlugin.java]: Dedent these lines.

File: runelite-client/src/main/java/net/runelite/client/plugins/menuentryswapper/MenuEntrySwapperConfig.java [hunk 3]
```diff
@@ -311,4 +311,14 @@ default boolean swapEnchant()
 	{
 		return true;
 	}
+
+	@ConfigItem(
+			keyName = "swapPvpAttack",
+			name = "PvP Attack",
+			description = "Swap Attack with Walk here when in PvP safe zones."
```
Comment (dekvall) [inline runelite-client/src/main/java/net/runelite/client/plugins/menuentryswapper/MenuEntrySwapperConfig.java]: Dedent these lines.

**Comment by nateprat** [comment]
With new update regarding MES, this will be deprecated

**Comment by Adam-** [comment]
I'm not convinced the new update rules this out. Also, the safe zones used to behave this way before they added in the ~5 second timer that makes safe zones pvp when you enter them right? So this seems fairly harmless. Does this deal with that timer (eg. not hide the attack options then)?

**Comment by deathbeam** [comment]
`Removes or deprioritises attack, cast (or similar) options from the minimenu in any scenario` I think its pretty clear?

**Comment by Adam-** [comment]
Ah hm, looked over that. That's probably referring to whenever you actually *can* attack but okay, fair enough.

**Comment by Nightfirecat** [comment]
[Per dekvall's comment](https://github.com/runelite/runelite/issues/10064#issuecomment-580506747) this feature is explicitly prohibited by Jagex now.

**Pull request closed without merging**